src/app/components/core/scheduler/list-schedules/list-schedules.component.ts

```diff
@@ -96,7 +98,7 @@ export class ListSchedulesComponent implements OnInit {
           //  filter by South and North categories name
           this.scheduleData = [];
           schedules.forEach(sch => {
-            if (!sn.includes(sch.name)) {
+            if (!sn.includes(sch.name) && sch.name.toUpperCase() !== 'FOGLAMP NOTIFICATION') {
```

Comment:
I think, filtering of schedules (e.g. south) is being handled at diff place

Reply:
please filter by process name `notification_c`. even for south and north, we can use filter schedules having process name [south, south_c, north, north_c] and avoid the call for categories